app/Statistics/Repository/IndividualRepository.php

```diff
@@ -109,9 +109,9 @@ private function commonGivenQuery(string $sex, string $type, bool $show_tot, int
                 // Exclude initials and particles.
                 if (!preg_match('/^([A-Z]|[a-z]{1,3})$/', $given)) {
                     if (\array_key_exists($given, $nameList)) {
-                        $nameList[$given] += (int) $count;
+                        $nameList[$given] += $count;
                     } else {
-                        $nameList[$given] = (int) $count;
+                        $nameList[$given] = $count;
```

Comment:
Surely these need to be cast from strings to integers.

The code that uses them calls `I18N::number()` which requires an integer.

Reply:
The value is already an integer

Reply:
I think it depends on the PDO driver.

I just tested it, and for SQLite, all columns (even numeric ones) are returned as strings.

Reply:
:(

Reply:
There may be other locations which need a casting too when used with SQLite.